site2/docs/security-tls-authentication.md

```diff
@@ -26,9 +26,10 @@ $ openssl pkcs8 -topk8 -inform PEM -outform PEM \
       -in admin.key.pem -out admin.key-pk8.pem -nocrypt
 ```
 
-Generate the certificate request. When asked for a **common name**, enter the **role token** which you want this key pair to authenticate a client as.
+Generate the certificate request. When asked for a **common name**, enter the **role token** which you want this key pair to authenticate a client as. 
 
 ```bash
+$ cp /etc/pki/tls/openssl.cnf ./
```

Comment:
hmm I think the documentation here was split from http://pulsar.apache.org/docs/en/security-tls-authentication/ . Hence there is an assumption that `openssl.cnf` was fetched in http://pulsar.apache.org/docs/en/security-tls-transport/#certificate-authority 

So I am not sure the fix is correct.